CHANGELOG.md

```diff
@@ -1,10 +1,59 @@
 # Change Log
 
+## [0.15.0](https://github.com/confluentinc/ksql/releases/tag/v0.15.0-ksqldb) (2021-01-20)
+
+### Highlights
+
+* expose support for array and struct keys ([#6722](https://github.com/confluentinc/ksql/pull/6722)) ([c7fc2b0](https://github.com/confluentinc/ksql/commit/c7fc2b0df7b64bcd0370054eb9c82d2880a2de53))
+* support PARTITION BY on multiple expressions ([#6803](https://github.com/confluentinc/ksql/pull/6803)) ([5a6b48e](https://github.com/confluentinc/ksql/commit/5a6b48efa63f0a0f007021c8cd73c71165b483fa))
+* ungate support for multi-column GROUP BY ([#6786](https://github.com/confluentinc/ksql/pull/6786)) ([9900623](https://github.com/confluentinc/ksql/commit/9900623bdc669ad46c0321ef9f4989454bcd1108))
+
+### Features

```

Comment:
We seem to be missing some changes. I don't see either https://github.com/confluentinc/ksql/pull/6694 or https://github.com/confluentinc/ksql/pull/6692 in this changelog. (I noticed since I'd have expected these two to accompany the other key enhancements in the highlights section above.) Any ideas what happened? LMK if not -- I'm happy to take a look since the reason changelog generation for 0.15 is a bit of a special snowflake is on me 😅

Reply:
I had to manually curate this one, so probably just missed those. I'll add those and take another pass to see if I've missed anything else.